source/hands_on/trex.rst

```diff
@@ -0,0 +1,82 @@
+TREX
+^^^^^^^
+
+Overview
+---------
+TREX (Trajectory-ranked Reward Extrapolation) was first proposed in `Extrapolating Beyond Suboptimal Demonstrations via Inverse Reinforcement Learning from Observations <https://arxiv.org/abs/1904.06387>`_, which uses ranked demonstrations to inference reward function. Different from previous methods, TREX seeks a reward function that explains the ranking over demonstrations, rather than justifying the demonstrations. Hence, this algorithm can learn good policies from highly suboptimal demostrations.
+
+Quick Facts

```

Comment:
Would you also comment on the final part of the paper, i.e. noisy ranking, ranking along with time